slang.h

```diff
@@ -1407,7 +1407,20 @@ extern "C"
     SLANG_API unsigned int spReflectionType_GetFieldCount(SlangReflectionType* type);
     SLANG_API SlangReflectionVariable* spReflectionType_GetFieldByIndex(SlangReflectionType* type, unsigned index);
 
+        /** Returns the number of elements in the given type.
+
+        This operation is valid for vector and array types. For other types it returns zero.
+
+        When invoked on an unbounded-size array it will return `SLANG_UNBOUNDED_SIZE`,
+        which is defined to be `size_t(-1)`.
```

Comment:
are you sure you want size_t(-1), as this could be 0x00000000ffffffff when perhaps ~size_t(0) is perhaps more the intention?

Reply:
Probably best to be safe, yes.